python/ppml/test/bigdl/ppml/fl/nn/pytorch/test_logistic_regression.py

```diff
@@ -49,7 +49,8 @@ def mock_process(data_train, target, client_id):
     set_one_like_parameter(model)
     loss_fn = nn.BCELoss()
     server_model = LogisticRegressionNetwork2()
-    ppl = Estimator.from_torch(client_model=model, 
+    ppl = Estimator.from_torch(client_model=model,
+                               client_id=client_id,
```

Comment:
The related client_id in another test party has already been set [here](https://github.com/intel-analytics/BigDL/blob/e1fae35cf75b3a4878a4306dedef0bc4976a675b/python/ppml/test/bigdl/ppml/fl/nn/pytorch/test_logistic_regression.py#L114)